### DISCUSSION TITLE
Urgent: Lost 2FA Access and Recovery Codes for npm Account (amankirmara)

### DISCUSSION THREAD

**Discussion by theamanone** [community/community#147769] [Other Feature Feedback, Questions, & Ideas]
Hello GitHub Community and npm Support,

I am writing to seek urgent assistance regaining access to my npm account, "amankirmara" (email: amankirmara143@gmail.com).

I have unfortunately lost access to my 2FA authentication method and no longer have the recovery codes. This is preventing me from logging in.

I have already tried the following:

Contacted npm support via email (support@npmjs.com). They responded by directing me to this forum.
Attempted the password reset process, but it requires either a security key or recovery code, neither of which I have.
Checked the npm support website (https://www.npmjs.com/support), which requires being logged in to access direct support.
To verify my ownership of the account, I can provide the following information:

Owned npm Package: I maintain the package socket-insight (npm install command: npm i socket-insight).
GitHub Account: My current GitHub username is "theamanone." My repositories related to socket-insight and other projects are located under this username.
Previous Email to npm Support: (Optional) You can briefly mention that you sent an email to support@npmjs.com on [Date of email] if you want to provide further context.
I understand the importance of 2FA and take full responsibility for losing my recovery information. I would be extremely grateful if someone from npm support or the community could assist me in regaining access to my account. I am willing to provide any further information necessary to verify my identity.

Thank you for your time and assistance.

Sincerely,

Aman Kirmara

**Comment by ebndev** [comment]
Hi @theamanone,

It looks like you may have been directed here incorrectly, sorry about that!

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). You can log in through your GitHub account.

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
